pkgs/development/interpreters/perl/default.nix

```diff
@@ -37,15 +37,11 @@ let
     patches =
       [
         # Do not look in /usr etc. for dependencies.
-        (if (versionOlder version "5.31.1") then ./no-sys-dirs-5.29.patch
```

Comment:
Should `no-sys-dirs-5.29.patch` be deleted as well?

Reply:
Thx! Forgot to remove the files as well. Fixed.